src/saturn_engine/worker/services/labels_propagator.py

```diff
@@ -0,0 +1,15 @@
+from saturn_engine.worker.executors.executable import ExecutableMessage
+
+from . import MinimalService
+
+
+class LabelsPropagator(MinimalService):
+    name = "labels_propagator"
+
+    async def open(self) -> None:
+        self.services.hooks.message_polled.register(self.on_message_polled)
+
+    async def on_message_polled(self, xmsg: ExecutableMessage) -> None:
+        xmsg.message.message.metadata[
```

Comment:
We not copy all labels? (And store it under `metadata["labels"]`?) Maybe even add a service option to define the list of labels to copy if we want to go fancy.

We also might want to do something like

```python
metadata_labels = xmsg.message.message.metadata.setdefault("labels", {})
for k, v in xmsg.queue.definition.labels.items():
  metadata_labels.setdefault(k, v)
```

so we don't overwrite existing message labels.